tests/test_name_resolution/good/toplevel_defs.catala_en

```diff
@@ -112,84 +112,129 @@ let glob2_10 = A {"y": glob1_2 >= 30., "z": 123. * 17.}
 let S2_6 (S2_in_11: S2_in) =
   decl temp_a_13 : any;
   try:
-    decl temp_a_16 : any;
-    let temp_a_16 (__17 : unit) =
-      return glob3_3 ¤44.00 + 100.;
+    decl temp_a_22 : any;
+    let temp_a_22 (__23 : unit) =
+      raise EmptyError;
+    decl temp_a_20 : any;
+    let temp_a_20 (__21 : unit) =
+      return false;
     decl temp_a_14 : any;
     let temp_a_14 (__15 : unit) =
-      return true;
-    temp_a_13 = handle_default [] temp_a_14 temp_a_16
+      decl temp_a_18 : any;
+      let temp_a_18 (__19 : unit) =
+        return glob3_3 ¤44.00 + 100.;
```

Comment:
This should use the localized printer!

Reply:
No, this is a debug intermediate representation output, the localised printer is intended for end-user directed values (like results of computations)